google_gke_infra/README.md

```diff
@@ -101,39 +105,39 @@ module "k8s" {
 }
 ```
 
-**NOTE:** All parameters are configurable. Please see [below](#variables) for information on each configuration option.
+>**NOTE:** You can configure all parameters. See [variables](#variables) for information on each configuration option.
 
-### Creating a Private Cluster
+### Create a private cluster
 
-By default, this creates a [Private GKE Cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters) where the nodes do not have public IP addresses. This is achieved with the `private_nodes` variable (see [optional-variables](#optional-variables)). The module deploys a [GCP Cloud NAT resource](https://cloud.google.com/nat/docs/overview?hl=en_US&_ga=2.47256615.-1497507305.1549638187) to enable public egress connectivity.
+By default, the basic module configuration creates a [Private GKE Cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters) where the Nodes do not have public IP addresses thanks to the )[**private_nodes**](#optional-variables) variable. The module deploys a [GCP Cloud NAT resource](https://cloud.google.com/nat/docs/overview?hl=en_US&_ga=2.47256615.-1497507305.1549638187) to enable public egress connectivity.
 
-### Using the Cluster as a TF Provider
+### Use the cluster as a Terraform provider
 
-In the same terraform file, you can use the cluster this module creates by adding the following resources to your `main.tf` file:
+In the same Terraform file, you can use the cluster created by modules by adding the following resources to your `main.tf` file:
```

Comment:
By `the same Terraform file` you mean the `main.tf` file? Or you mean `in one terraform file you can have the configuration for the private cluster and also the config which allows using this cluster as a Terraform provider`? In such a case I would use "in one file/ in a single file".

Reply:
Yes, the `main.tf` or whichever terraform file the downstream user has defined their modules in (any `*.tf` file in the same folder will do)